dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -505,7 +525,7 @@ CDefaultContextMenu::AddStaticContextMenusToMenu(
         /* By default use verb for menu item name */
         mii.dwTypeData = (LPWSTR)info.Verb.GetString();
 
-        WCHAR wszKey[256];
+        WCHAR wszKey[sizeof("shell\\") + MAX_VERB];
```

Comment:
side note: in principle this sizeof is really a `_countof` (of characters), but ofc. it's going to generate the same value here (since the string is ansi)